### PR TITLE
Add QRCode error.

### PR DESCRIPTION
**Pull request by SAnatoliiS**

### CODE REVIEW

File: packages/chats/src/BubbleQRCode.tsx [hunk 1]
```diff
@@ -1,20 +1,86 @@
...
 export const ChatBubbleQRCode: React.FC<ChatQRCodeMessage> = (message: ChatQRCodeMessage) => {
     return <BubbleQRCode {...message} />;
 };
 
+const getErrorMessage = (messageStatus: MessageStatus): string => {
+    switch (messageStatus) {
+        case MessageStatus.Received:
+            return uiLocalized.Chats.QRCode.errorReceived;
+        default:
+            return uiLocalized.Chats.QRCode.errorSended;
```
Comment (sertony) [inline packages/chats/src/BubbleQRCode.tsx]:
```suggestion
            return uiLocalized.Chats.QRCode.errorSent;
```

Comment (SAnatoliiS) [reply packages/chats/src/BubbleQRCode.tsx]: 😅

**Review by sertony — APPROVED** [review]

File: packages/chats/src/BubbleQRCode.tsx [hunk 2]
```diff
@@ -1,20 +1,86 @@
...
+const getErrorMessage = (messageStatus: MessageStatus): string => {
+    switch (messageStatus) {
+        case MessageStatus.Received:
+            return uiLocalized.Chats.QRCode.errorReceived;
+        default:
+            return uiLocalized.Chats.QRCode.errorSent;
+    }
+};
+
+const renderIcon = (
```
Comment (savelichalex) [inline packages/chats/src/BubbleQRCode.tsx]: Maybe call it `renderErrorIcon`? it's confusing now :)

**Review by savelichalex — APPROVED** [review]
Just a small naming concern, overall seems good to me :)

**Pull request merged**